--- libseccomp-2.0.0/Makefile~	2013-01-16 21:07:08.000000000 +0100
+++ libseccomp-2.0.0/Makefile	2013-01-28 17:51:12.258014973 +0100
@@ -85,7 +85,8 @@
 
 install: $(SUBDIRS_BUILD)
 	@$(ECHO_INFO) "installing in $(INSTALL_PREFIX) ..."
-	$(INSTALL_PC_MACRO) libseccomp.pc
+	$(INSTALL) -d $(INSTALL_LIB_DIR)/pkgconfig
+	$(INSTALL) libseccomp.pc $(INSTALL_LIB_DIR)/pkgconfig
 	@for dir in $(SUBDIRS_INSTALL); do \
 		$(ECHO_INFO) "installing from $$dir/"; \
 		$(MAKE) -C $$dir install; \
