--- libseccomp-1.0.0/Makefile.orig	2012-07-28 00:39:00.000000000 +0200
+++ libseccomp-1.0.0/Makefile	2012-10-27 21:21:08.860539706 +0200
@@ -85,7 +85,8 @@
 
 install: $(SUBDIRS_BUILD)
 	@$(ECHO) ">> INFO: installing in $(INSTALL_PREFIX) ..."
-	$(INSTALL_PC_MACRO) libseccomp.pc
+	$(INSTALL) -d $(INSTALL_LIB_DIR)/pkgconfig
+	$(INSTALL) libseccomp.pc $(INSTALL_LIB_DIR)/pkgconfig
 	@for dir in $(SUBDIRS_INSTALL); do \
 		$(ECHO) ">> INFO: installing from $$dir/"; \
 		$(MAKE) -C $$dir install; \
